app/views/accounts/actions/_regenerate_backup_codes.html.erb

```diff
@@ -1,3 +1,6 @@
-<%= link_to backup_code_regenerate_path, class: 'account-action-button' do %>
-  <%= prefix_with_plus(t('forms.backup_code.regenerate')) %>
-<% end %>
+<%= render ButtonComponent.new(
+      action: ->(**tag_options, &block) do
+        link_to(backup_code_regenerate_path, **tag_options, &block)
+      end,
+      icon: :add,
```

Comment:
We'll want classes like we have with others:

```suggestion
      icon: :add,
      class: 'usa-button usa-button--outline margin-top-2',
```

Though, in fact, `usa-button` is default, and `outline` can be passed as an option:

```suggestion
      icon: :add,

```suggestion
      icon: :add,
      outline: true,
      class: 'margin-top-2',
```

![image](https://user-images.githubusercontent.com/1779930/154744880-bdf39022-44b4-4212-b935-3959d50760a2.png)